File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/02.Conditional-Statements/03-additional/02.sleepyTomCat.js
function sleepyTomCat(input) {

    const daysOfTheYear = 365;
    const normForPlay = 30000;
    const playInWorckingDay = 63;
    const playInRestDay = 127;

    const restDay = Number(input[0]);

    const workingDay = daysOfTheYear - restDay;
    const totalTimeForPlay = workingDay * playInWorckingDay + restDay * playInRestDay;
    const difference = Math.abs(normForPlay - totalTimeForPlay);
    const hours = Math.floor(difference / 60);
    const minutes = difference % 60;

    if (totalTimeForPlay > normForPlay) {
        console.log("Tom will run away");
        console.log(`${hours} hours and ${minutes} minutes more for play`);
    } else {
        console.log("Tom sleeps well");
        console.log(`${hours} hours and ${minutes} minutes less for play`);
    }
}
sleepyTomCat(["113"]);

File: Python_Basics/Homeworks-And-Labs/02.conditional_statements/03.additional/02.sleepy_tom_cat.py
daysOfTheYear = 365
normForPlay = 30000
playInWorkingDay = 63
playInRestDay = 127

restDay = int(input())

workingDay = daysOfTheYear - restDay
totalTimeForPlay = workingDay * playInWorkingDay + restDay * playInRestDay
difference = abs(normForPlay - totalTimeForPlay)
hours = difference // 60
minutes = difference % 60

if totalTimeForPlay > normForPlay:
    print("Tom will run away")
    print(f'{hours} hours and {minutes} minutes more for play')
else:
    print("Tom sleeps well")
    print(f'{hours} hours and {minutes} minutes less for play')
